modules/persistent-storage-rhv.adoc

```diff
@@ -0,0 +1,65 @@
+[id="persistent-storage-rhv_{context}"]
+= Creating a persistent volume on RHV
+
+When you create a PersistentVolumeClaim(PVC) object, OpenShift Container Platform provisions a new persistent volume (PV) and creates a PersistentVolume object.
+Prerequisites
+You are logged in to a running OpenShift Container Platform cluster.
+Correct RHV credentials in  “ovirt-credentials” secret.
+oVirt CSI Driver installed.
+At least one Storage class defined.
+Procedure (UI)
+To dynamically create a persistent volume on RHV using the web console:
+In the OpenShift Container Platform console, click Storage → Persistent Volume Claims.
+In the persistent volume claims overview, click Create Persistent Volume Claim.
+Define the required options on the resulting page.
+Select the appropriate StorageClass (ovirt-csi-sc by default).
+Enter a unique name for the storage claim.
+Select the access mode, currently RWO is the only supported access mode.
+Define the size of the storage claim.
+Select the Volume Mode:
+Filesystem - mounted into Pods as a directory (default)
+Block - block device, without any filesystem on it.
+Click Create to create the PersistentVolumeClaim and generate a PersistentVolume.
+
+Procedure (CLI)
+To dynamically create a RHV CSI volume using the command-line interface (CLI):
+. Create and save a file with the `PersistentVolumeClaim` object described by the following YAML:
++
+.pvc-ovirt.yaml
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-ovirt
+spec:
+      storageClassName: ovirt-csi-sc [1]
+      accessModes:
+          - ReadWriteOnce
+      Resources: [2]
+          requests:
+              storage: 100Gi
+      volumeMode: Filesystem | Block [4]
+----
++
+<1> Name of the required storage class.
+<2> Volume size
```

Comment:
resources? instead of volume size?

Reply:
This is a technical question that I don't have an answer for...